docs/events/api/timeline-template-api-update.asciidoc

```diff
@@ -0,0 +1,385 @@
+  "version": "WzE5MTcsMV0=" <4>
+}
+--------------------------------------------------
+<1> `templateTimelineId` must match the existing Timeline template's `templateTimelineId`. 
+<2> `templateTimelineVersion` must be a Timeline template version greater than the existing one.
+<3> `timelineId` must match the existing Timeline's `savedObjectId`.
+<4> `version` must match the existing Timeline version.
+
+==== Response code
+
+`200`::
+    Indicates a successful call.
+
+==== Response payload
+
+A JSON Timeline object with a unique `savedObjectId` and its `version`.
+
+*Example 1*
+
+Update Timeline response payload:
+
+[source,json]
+--------------------------------------------------
+{
+  "data": {
+    "persistTimeline": {
+      "code": 200,
+      "message": "success",
+      "timeline": {
+        "savedObjectId": "4bc294e0-3516-11ee-9f62-49614d8a84fd",
+        "version": "WzE5MTgsMV0=",
+        "columns": [
+          {
+            "columnHeaderType": "not-filtered",
+            "id": "@timestamp"
+          },
+          {
+            "columnHeaderType": "not-filtered",
+            "id": "message"
+          },
+          {
+            "columnHeaderType": "not-filtered",
+            "id": "event.category"
+          },
+          {
+            "columnHeaderType": "not-filtered",
+            "id": "event.action"
+          },
+          {
+            "columnHeaderType": "not-filtered",
+            "id": "host.name"
+          },
+          {
+            "columnHeaderType": "not-filtered",
+            "id": "source.ip"
+          },
+          {
+            "columnHeaderType": "not-filtered",
+            "id": "destination.ip"
+          },
+          {
+            "columnHeaderType": "not-filtered",
+            "id": "user.name"
+          }
+        ],
+        "dataProviders": [],
+        "dataViewId": null,
+        "description": "",
+        "eventType": "all",
+        "excludedRowRendererIds": [],
+        "favorite": [],
+        "filters": [],
+        "kqlMode": "filter",
+        "kqlQuery": {
+          "filterQuery": null
+        },
+        "title": "abd",
+        "templateTimelineId": null,
+        "templateTimelineVersion": null,
+        "dateRange": {
+          "start": 1587370079200,
+          "end": 1587456479201
+        },
+        "savedQueryId": null,
+        "created": 1587468588922,
+        "createdBy": "casetester",
+        "updated": 1691408201273,
+        "updatedBy": "elastic",
+        "timelineType": "default",
+        "status": "active",
+        "sort": [
+          {
+            "sortDirection": "desc",
+            "columnId": "@timestamp"
+          }
+        ],
+        "eventIdToNoteIds": [],
+        "noteIds": [],
+        "notes": [],
+        "pinnedEventIds": [],
+        "pinnedEventsSaveObject": []
+      }
+    }
+  }
+}
+--------------------------------------------------
+
+*Example 2*
+
+Update Timeline template response payload:
```

Comment:
Suggesting some minor tweaks for clarity here - ofc the edits are totally optional.

```suggestion
Response payload of an updated Timeline template:
```